src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1021,52 +1045,36 @@ def test_WINNF_FT_S_FDB_6(self, config_filename):
 
     # Step 9: Send Grant Request 'G2'
     # Forming of 'G2' grants for all the configured grants
-    addCbsdIdsToRequests(cbsd_ids,config['grantRequests'][1])
+    addCbsdIdsToRequests(cbsd_ids, config['grantRequests'][1])
     grant_request_g2 = {'grantRequest': config['grantRequests'][1]}
 
     grant_response_g2 = self._sas.Grant(grant_request_g2)['grantResponse']
 
     # Check the grant response code is as expected (SUCCESS or INTERFERENCE).
-    for resp in grant_response_g2:
-      self.assertIn(resp['response']['responseCode'], config['expectedResponseCodes'][1])
+    for resp_num, resp in enumerate(grant_response_g2):
+      self.assertIn(resp['response']['responseCode'], config['expectedResponseCodes'][1][resp_num])
 
     del grant_request_g2, grant_response_g2
 
   def generate_FDB_7_default_config(self, filename):
     """Generates the WinnForum configuration for FDB.7"""
 
     # Load devices info
-    device_a = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_b.json')))
-    device_b = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_a_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_a_1 = device_a_0
```

Comment:
Why are you using the same device twice? Why not just use device_c?

Reply:
Updated,   device 'c' is used now